core/src/main/java/hudson/model/UnprotectedRootAction.java

```diff
@@ -32,4 +32,26 @@
  * @since 1.398
  */
 public interface UnprotectedRootAction extends RootAction, ExtensionPoint {
```

Comment:
@jglick it makes a difference when it comes to security. see [Jenkins.java#L3826-L3829](https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/jenkins/model/Jenkins.java#L3826-L3829).

normal `RootAction`s require a leading slash AFAICT.